### PR TITLE
Support text properties

### PR DESCRIPTION
**Pull request by justinvdm**
We need to be able to do this:

```
### rar
foo: `bar baz`
baz: `
  quux
    corge
      grault
`
garply[text]: `
  waldo
    fred
`
```

### CODE REVIEW

**Comment by justinvdm** [comment]
Depends on #5, otherwise ready for review.

**Comment by justinvdm** [comment]
Ready for review.

File: src/parser.pegjs [hunk 1]
```diff
@@ -108,15 +105,29 @@ lcletter 'lower case letter'
 
 
 text 'text'
+  = [^`]*
```
Comment (hodgestar) [inline src/parser.pegjs:127]: Do we want to allow people to include "`" in their text via an escape of some sort? Doesn't seem like it'll be needed that often, but would be nice to be able to cope with all strings?

File: tests/numl.test.js [hunk 2]
```diff
@@ -374,4 +380,63 @@ describe("numl", function() {
...
+      sequences: [{
+        blocks: [{
+          properties: {
+            foo: {
+              __type__: 'text',
+              value: 'bar baz',
+            },
+            baz: {
+              __type__: 'text',
+              value: str`
```
Comment (hodgestar) [inline tests/numl.test.js:448]: I don't even want to know why Javascript language writers thought this feature was necessary. :)

**Comment by hodgestar** [comment]
Left one question, otherwise looks good. :)

File: src/parser.pegjs (hunk 1 above)
Comment (justinvdm) [inline src/parser.pegjs:127]: +1, to get the PR backlog out the way, could I make this a ticket for now?

Comment (hodgestar) [inline src/parser.pegjs:127]: New ticket for this is perfect.

**Comment by hodgestar** [comment]
Hmm. Branch has merge conflicts (I'm guessing its the built `numl.js` file).

**Comment by justinvdm** [comment]
Yep, conflicts with #6, merge conflicts fixed.

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**